src/dbSta/src/dbSta.tcl

```diff
@@ -85,5 +85,55 @@ proc sta_warn { id msg } {
   utl::warn STA $id $msg
 }
 
-# namespace
+rename report_units report_units_raw
+
+proc report_units { args } {
+
+  report_units_raw
+
+  utl::push_metrics_stage "run__flow__platform__{}_units"
+
+  foreach unit {{"time" "timing"} {"power" "power"} {"distance" "distance"}} {
+    set utype [lindex $unit 0]
+    set umetric [lindex $unit 1]
+    utl::metric "$umetric" "[unit_suffix $utype]"
+  }
+
+  utl::pop_metrics_stage
+}
+
+rename report_tns report_tns_raw
```

Comment:
This relies on the sta commands being present when this code is executed.  Has this been tested?

Its a bit ugly but I don't have a better idea that doesn't pollute sta/